Backpack-SwiftUI/SearchInputSummary/Classes/BPKSearchInputSummary.swift

```diff
@@ -93,6 +94,12 @@ public struct BPKSearchInputSummary: View {
         }
     }
     
+    @ViewBuilder
+    private var placeholderView: Text {
+        Text(placeholder)
+            .foregroundColor(Color(BPKColor.textSecondaryColor.value))
```

Comment:
Should you not be using a BPKText here?
```swift
BPKText(placeholder).foregroundColor(.textSecondary)

Reply:
No, it's accepted only Text view. It won't compile with BPKText